pkg/utils/utils.go

```diff
@@ -64,6 +64,9 @@ func GetDefaultCmsisPackRoot() (root string) {
 	}
 	if root == "" {
 		root = os.Getenv("HOME")
+		if root != "" {
+			root = filepath.Clean(root + "/.cache")
+		}
 	}
 	if root != "" {
 		root = filepath.Clean(root + "/Arm/Packs")
```

Comment:
We should also set the right capitalization according to the OS `Arm/Packs` or `arm/packs`.

Reply:
Ahh, you are right, Overlooked this.
[Addressed review comments](https://github.com/Open-CMSIS-Pack/cbuild/pull/14/commits/90762ca37776f8516319a36c335227db6aaaae71)

Reply:
For being coherent with the [`CrossPlatformUtils::GetDefaultCMSISPackRootDir()`](https://github.com/Open-CMSIS-Pack/devtools/blob/main/libs/crossplatform/src/CrossPlatformUtils.cpp#L34) I would propose something like this, would you agree?
``` go
func GetDefaultCmsisPackRoot() (root string) {
	if runtime.GOOS == "windows" {
		root = os.Getenv("LOCALAPPDATA")
		if root == "" {
			root = os.Getenv("USERPROFILE")
			if root != "" {
				root = root + "\\AppData\\Local"
			}
		}
		if root != "" {
			root = root + "\\Arm\\Packs"
		}
	} else {
		root = os.Getenv("XDG_CACHE_HOME")
		if root == "" {
			root = os.Getenv("HOME")
			if root != "" {
				root = root + "/.cache"
			}
		}
		if root != "" {
			root = root + "/arm/packs"
		}
	}
	return filepath.Clean(root)
}
```

Reply:
I agree. Thanks. Its been updated